### PR TITLE
Fix tryKubernetes button

### PR DESCRIPTION
**Pull request by imbsky**
Fix tryKubernetes button.

| Before | After |
| :-----: | :----: |
| before | after |

### CODE REVIEW

File: sass/_base.sass [hunk 1]
```diff
@@ -334,7 +334,8 @@ ul.global-nav
 
 .open-nav, .y-enough
 	#tryKubernetes
-		width: 150px
+		width: initial
```
Comment (kbhawkey) [inline sass/_base.sass]: See issue #10309 and PR #10525.
What about using width: auto?

**Comment by tfogo** [comment]
Thank you for your contribution @imbsky!

/lgtm
/approve

**Comment by imbsky** [comment]
You're welcome!

**Pull request merged**